Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3'
+# gem 'sqlite3'
+gem 'pg', '~> 0.20.0'
```

Comment:
@fabriziobertoglio1987 Best way is probably as you suggested. Have both configurations in the database.yml file, with one commented out. Also include gems for both, so *both* `gem 'pg', '~> 0.20.0'` and `gem 'sqlite3'` in the Gemfile.